client/my-sites/subscribers/components/empty-list-view/empty-list-view.tsx

```diff
@@ -0,0 +1,50 @@
+import { Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { GrowAudienceIcon, ImportSubscribersIcon, TurnIntoSubscribersIcon } from './icons';
+import './styles.scss';
+
+type EmptyListCTALinkProps = {
+	href: string;
+	icon: JSX.Element;
+	text: string;
+};
+
+const EmptyListCTALink = ( { href, icon, text }: EmptyListCTALinkProps ) => (
```

Comment:
I wonder if we could reuse some of the existing components from [devdocs](http://calypso.localhost:3000/devdocs/design).

I see these for instance, but not sure if they will be suitable for our case:

| Card | Header |
|--------|--------|
| ![Markup on 2023-06-07 at 12:48:48](https://github.com/Automattic/wp-calypso/assets/25105483/8c8ee973-4a49-4426-8e02-dae206c37c80) | ![Markup on 2023-06-07 at 12:49:22](https://github.com/Automattic/wp-calypso/assets/25105483/d2ac82c6-558d-4fca-b70d-11385f59ce50) | 

@crisbusquets 👋🏼 Do you know if these tabs from the design belong to an existing component? Thank you.

![Markup on 2023-06-07 at 13:03:11](https://github.com/Automattic/wp-calypso/assets/25105483/7528f5d6-00a3-4c19-af10-ca61e6b2fd8e)

Reply:
Please check also with @daledupreez as their team is implementing similar lists elsewhere, to ensure we're sharing components.

Reply:
Thank you Cris & Mikael!

> The main difference between both is the padding (16 vs 24) and that the one in the onboarding flow also has a paragraph.

> Please check also with @daledupreez as their team is implementing similar lists elsewhere, to ensure we're sharing components.

Nice! The components used in the Newsletter flow look pretty similar, so I think we could probably reuse them (we could just adjust the padding in a custom CSS ant not use the description).

Maybe Dale has a different perspective.

Reply:
In case it's helpful, I've added developer handoff information in this [Figma file](HSljkKnYECEEegL3DqP2xC-fi-9167_88759) 🙏🏼

Reply:
In case it's helpful, I've added dev handoff instructions in this [Figma file](h8tMpJlCqNFemEerU9owHI-fi-2173_68347) 🙏🏼

You can compare it with the Newsletter onboarding flow block, [which is really similar](HSljkKnYECEEegL3DqP2xC-fi-9167_88759).

Reply:
I have updated the implementation to utilize `Card`/`CardBody` components from `wordpress/components`, which are a closer match to the UI requirements. With that, most of the customizations were specific to the Subscribers screen. If there are more generic components available that support additional features, we can migrate to them in a follow-up PR. Thank you for the feedback, let me know if you have any other thoughts.

Reply:
> https://github.com/orgs/Automattic/teams/serenity, what state are the current UI components in? Can we use them for this use case?

I have created a dedicated task for replacing the `Card` component: https://github.com/Automattic/wp-calypso/issues/77968 (once we get more info whether the new component is ready).

If the component is not ready by the time we get to that task, we can create it in @Automattic/apex.